atomic-design.md

```diff
@@ -0,0 +1,101 @@
+# Atomic Design Guide
+
+> Helpful guidance and tips for building front-end components atomically.
+
+## What is Atomic Design?
+
+Created by [Brad Frost](http://bradfrost.com/blog/post/atomic-web-design/), Atomic Design is a system for building and maintaining reusable components (we refer to them as elements) for a web platform. By creating simple components, and combining them to build larger and more complex components, we can ensure a consistent design throughout our projects.
```

Comment:
It's confusing because Components are also the individual CF packages. I've updated updated to refer to code chunks as elements and only refer to the CF Components as components.